common/emu_constants.cpp

```diff
@@ -380,5 +391,51 @@ std::string EQ::constants::GetEnvironmentalDamageName(uint8 damage_type)
 		auto damage_types = EQ::constants::GetEnvironmentalDamageMap();
 		return damage_types[damage_type];
 	}
+
 	return std::string();
 }
+
+const std::map<uint8, std::string>& EQ::constants::GetStuckBehaviorMap()
+{
+	static const std::map<uint8, std::string> stuck_behavior_map = {
+		{ StuckBehavior::RunToTarget, "Run To Target" },
+		{ StuckBehavior::WarpToTarget, "Warp To Target" },
+		{ StuckBehavior::TakeNoAction, "Take No Action" },
+		{ StuckBehavior::EvadeCombat, "Evade Combat" }
+	};
+
+	return stuck_behavior_map;
+}
+
+std::string EQ::constants::GetStuckBehaviorName(uint8 behavior_id)
+{
+	if (EQ::ValueWithin(behavior_id, StuckBehavior::RunToTarget, StuckBehavior::EvadeCombat)) {
+		auto stuck_behaviors = EQ::constants::GetStuckBehaviorMap();
+		return stuck_behaviors[behavior_id];
+	}
+
+	return std::string();
+}
+
+const std::map<uint8, std::string>& EQ::constants::GetSpawnAnimationMap()
+{
+	static const std::map<uint8, std::string> spawn_animation_map = {
+		{ SpawnAnimations::Standing, "Standing" },
+		{ SpawnAnimations::Sitting, "Sitting" },
+		{ SpawnAnimations::Crouching, "Crouching" },
+		{ SpawnAnimations::Laying, "Laying" },
+		{ SpawnAnimations::Looting, "Looting" }
+	};
+
+	return spawn_animation_map;
+}
+
+std::string EQ::constants::GetSpawnAnimationName(uint8 animation_id)
+{
+	if (EQ::ValueWithin(animation_id, SpawnAnimations::Standing, SpawnAnimations::Looting)) {
+		auto spawn_animations = EQ::constants::GetSpawnAnimationMap();
```

Comment:
Any reason we didn't just inline this with the return

Reply:
Not sure what you mean here. Glad to do whatever cleans it up, just need clarification.

Reply:
`return EQ::constants::GetSpawnAnimationMap()[animation_id]`

Haven't tested, but just inlines without declaring

Reply:
Makes sense, guess I could clean up the others to do that, assuming it works.

Reply:
Converted them to use .find on the map itself.
Example:
```cpp
return EQ::constants::GetLDoNThemeMap().find(theme_id)->second;
```